test_wrapper/Dockerfile

```diff
@@ -28,14 +26,14 @@ RUN mkdir CFAndroidSDK && cd ./CFAndroidSDK && \
     git checkout "$WRAPPERS_BRANCH" && \
     git submodule init && git submodule update && \
     echo "sdk.dir=/Android" > local.properties && \
-    echo "{\"selfTest\": $SELF_TEST, \"port\": $PORT, \"apiKey\": \"$API_KEY\", \"logger\": \"filesystem\"}" \
-        > "test_wrapper/wrapper.json" && \
-        echo "wrapper.json:" && cat "test_wrapper/wrapper.json" && \
     ../gradle-7.0/bin/gradle wrapper --gradle-version 7.0 && ../gradle-7.0/bin/gradle --stop
 
 EXPOSE $PORT
 
 CMD cd ./CFAndroidSDK && git fetch && git pull && \
+    echo "{\"selfTest\": $SELF_TEST, \"port\": $PORT, \"sdkKey\": \"$SDK_KEY\", \"logger\": \"filesystem\", \"enableStreaming\": $ENABLE_STREAMING, \"eventUrl\": \"$EVENT_URL\", \"sdkBaseUrl\": \"$SDK_BASE_URL\"}" \
```

Comment:
will the sdkKey part of this just create an empty entry?

Reply:
It should not. It will populate with the environment variable value of the SDK_KEY.

Reply:
yes, you are right! my bad